mmedit/datasets/builder.py

```diff
@@ -32,11 +46,16 @@ def build_dataset(cfg, default_args=None):
     Returns:
```

Comment:
You may want to update the doc str, also add doc str to _concat_dataset

Reply:
Though I think for this very function the current docstring is enough, I've added more explanation to the possible behavior of the function.